src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -901,6 +901,181 @@ BOOST_AUTO_TEST_CASE(GradientFastLSTMLayerTest)
   BOOST_REQUIRE_LE(CheckGradient(function), 0.2);
 }
 
+/**
+ * Testing the overloaded Forward of the LSTM layer.
+ * for retrieving the Cell State. Besides output
+ * the LSTM the overloaded function provides read
```

Comment:
I think it should be like `Besides output, the overloaded LSTM Forward() function...`

Reply:
Okay, rectifiied.